applications/serial_lte_modem/Kconfig

```
@@ -81,8 +81,16 @@ config SLM_START_SLEEP
 	help
 	  Put nRF9160 into deep sleep mode after startup.
 
-config SLM_INTERFACE_PIN
-	int "Interface GPIO to wake up or exit idle mode"
+config SLM_WAKEUP_PIN
+	int "Interface GPIO to exit from sleep or idle mode"
+
+config SLM_INDICATE_PIN
+	default -1
+	int "Interface GPIO to indicate data available or unexpected reset"
```

it's a good custom to have the type (and title) of the setting at top.

Also, the title is a bit long, some of the title more belongs in the help text of the setting.

Proposal:
```suggestion
	int "SLM indication pin"
	default -1
	help
	  Interface GPIO pin used by Serial LTE Modem to indicate when data is available or an unexpected reset has occured
```